libs/martech/martech.js

```diff
@@ -118,14 +118,14 @@ export const getTargetPersonalization = async () => {
   const responseStart = Date.now();
   window.addEventListener(ALLOY_SEND_EVENT, () => {
     const responseTime = calculateResponseTime(responseStart);
-    window.lana.log(`target response time: ${responseTime}`, { tags: 'errorType=info,module=martech' });
+    window.lana.log(`target response time: ${responseTime}`, { tags: 'module-martech', errorType: 'i' });
```

Comment:
is `module-martech` tag existing? wouldn't `martech` be clear enough, yet smaller?

Reply:
yes, I will make this change.